### PR TITLE
support for xiaomi aqara thermometer and few other fixes

### PR DESCRIPTION
**Pull request by hobbypunk90**

### CODE REVIEW

**Comment by bbbenji** [comment]
Looks good, thank you.

File: config.yaml.example [hunk 1]
```diff
@@ -2,6 +2,7 @@ mqtt:
   host: 192.168.1.1
   username: user
   password: password
+  topic_prefix: hostname
```
Comment (zewelor) [inline config.yaml.example]: Maybe we should also add port here, if its added in code, just to show its possible to specify it here, without reading mqtt.py code.

Comment (hobbypunk90) [reply config.yaml.example]: @zewelor you are right

**Comment by zewelor** [comment]
Looks great, thanks for contribution !

**Comment by bbbenji** [comment]
Assuming you are using Home-Assistant, could you please update the Wiki with an example HASS config: https://github.com/zewelor/bt-mqtt-gateway/wiki/Home-Assistant

**Comment by hobbypunk90** [comment]
i add an example, when i switch to bt-mqtt-gateway 😁 
the next days i'll try it with the mifora sensors 😉 i hope i can help there a little bit 🙂

**Comment by bbbenji** [comment]
It’s really appreciated:)

**Comment by bbbenji** [comment]
@hobbypunk90 could you please explain the changes made here: https://github.com/zewelor/bt-mqtt-gateway/pull/18/commits/6dae82ba2269bfac76e70377c80203a811755f87

Another user has made the following changes to the same lines in order to get it working on older pip: https://github.com/jumping2000/bt-mqtt-gateway/commit/cfed80054270de5b5e247419164327eb1077ccee

Now we are wondering what is the correct way to go about this.

**Comment by hobbypunk90** [comment]
@bbbenji that's the point the changes of @jumping2000 don't work an an environment with python 3.7 (not sure if 3.6 also) with pip greater than 10, to only option to support both is to check the pip version and use the one way for older and the other for newer pip versions.

i have to say, i only searched and copied the solution for working with pip on newer version 😉

**Comment by hobbypunk90** [comment]
the "best" solution looks like this i think:
```python
from pip import __version__ as pip_version
if pip_version > "10.0":
  from pip._internal import main as pip_main
else:
  from pip import main as pip_main
```
i only checked it on ipython but i think this should work

**Comment by bbbenji** [comment]
Thanks for explaining! This whole project was made by just searching and copying solutions :smile:

**Pull request merged**